src/Surfnet/Stepup/Identity/Event/PhonePossessionProvenEvent.php

```diff
@@ -173,4 +185,14 @@ public function setSensitiveData(SensitiveData $sensitiveData)
         $this->commonName  = $sensitiveData->getCommonName();
         $this->phoneNumber = $sensitiveData->getSecondFactorIdentifier();
     }
+
+    public function obtainUserData(): array
+    {
+        $serializedPublicUserData = $this->serialize();
+        $serializedSensitiveUserData = $this->getSensitiveData()->serialize();
+        $serializedCombinedUserData = array_merge($serializedPublicUserData, $serializedSensitiveUserData);
+        $whitelist = array_flip(self::$whitelist);
+        return array_intersect_key($serializedCombinedUserData, $whitelist);
+//        return $serializedCombinedUserData;
```

Comment:
A dev left over? Please remove the stray commented line of code!

Reply:
aremoved